catroid/src/main/java/org/catrobat/catroid/content/actions/ArduinoSendDigitalValueAction.java

```diff
@@ -66,12 +66,7 @@ protected void begin() {
 					interpretationException);
 		}
 
-		if ((!restart)
-				&& (pinNumberInterpretation == 2)
-				|| (pinNumberInterpretation == 4)
-				|| (pinNumberInterpretation == 8)
-				|| (pinNumberInterpretation == 12)
-				|| (pinNumberInterpretation == 13)) {
+		if (!restart) {
 			this.pin = pinNumberInterpretation;
 			this.value = pinValueInterpretation;
 		}
```

Comment:
also notice that because of the changes I made to portValue in my last commit, [#1953](https://github.com/Catrobat/Catroid/pull/1953) will result in a merge conflict if merged after this pull request.